app/models/manageiq/providers/openstack/cloud_manager.rb

```diff
@@ -87,6 +94,10 @@ def supports_provider_id?
     true
   end
 
+  def supports_cinder_service?
+    true
```

Comment:
Unless this is a very early deployment of OpenStack (which may be possible, but maybe we don't support that?) then OpenStack should *support* cinder always, shouldn't it? (and the same holds true for Swift)